src/libraries/Builder/Microsoft.Agents.Builder/App/AdaptiveCards/AdaptiveCardInvokeResponseFactory.cs

```diff
@@ -16,7 +16,7 @@ public static class AdaptiveCardInvokeResponseFactory
         /// </summary>
         /// <param name="adaptiveCard">An Adaptive Card.</param>
         /// <returns>The response that includes an Adaptive Card that the client should display.</returns>
-        public static AdaptiveCardInvokeResponse AdaptiveCard(AdaptiveCard adaptiveCard)
+        public static AdaptiveCardInvokeResponse AdaptiveCard(global::AdaptiveCards.AdaptiveCard adaptiveCard)
```

Comment:
global:: is scary... we missing a using here?

Reply:
Weird.  Checking.